src/components/footer/footer.module.css

```diff
@@ -30,6 +30,11 @@
   cursor: pointer;
 }
 
+.socialIcon {
+  display: flex;
+  align-items: center; /* Ensures vertical alignment */
```

Comment:
thanks, we can probably remove the comment here, the property should hopefully speak for itself